### PR TITLE
Added Solution for Pythagorean Triplet

### PR DESCRIPTION
**Pull request by Rohit-RA-2020**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,7 +191,7 @@
 |------|:----------:|:--------:|
 | [Combo Meals](https://www.hackerrank.com/challenges/combo-meal) | Easy | |
 | [Difference and Product](https://www.hackerrank.com/challenges/difference-and-product) | Easy | |
-| [Pythagorean Triplet](https://www.hackerrank.com/challenges/pythagorean-triple) | Easy | |
+| [Pythagorean Triplet](https://www.hackerrank.com/challenges/pythagorean-triple) | Easy|[![cpp](https://img.icons8.com/color/35/000000/c-plus-plus-logo.png)](cpp/pythagoreanTriplet.cpp) | |
```
Comment (anishLearnsToCode) [inline README.md]: Thanks for solving this 💯 when you add the image logo link make sure to add it in the `| |` which is already there as currently there is an extra `| |` after the link. 

also when you solve a question in any language - this case cpp, update the badge in the `README.md` file as well. 

so `problems-solved-cpp` --> `problems-solved-cpp`

Also update the Problems solved badge if the you are solving a new problem that wasn't solved before. 
`problems-solved` --> `problems-solved`

Thanks again 😃

File: cpp/pythagoreanTriplet.cpp [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+using namespace std;
+
+int main()
```
Comment (anishLearnsToCode) [inline cpp/pythagoreanTriplet.cpp]: The indentation style that you use is called Allman style which is like
```
function a()
{
    body
    body
}
```

Prefer instead to use the K & R style:
```
function a() {
    body
    body
}
```

the k & r style is favoured in industry.

**Comment by Rohit-RA-2020** [comment]
Resolved mentioned issues

**Comment by anishLearnsToCode** [comment]
Thanks for implementing my comments 🥳

**Comment by anishLearnsToCode** [comment]
Thank you for addressing the comments! 🥳 🥳  

This branch is now merged and the following comments don't apply, I resolved the conflicts myself but still thought you should have this.

okay now you will learn `git rebase`, an excellent thing that is required in actual software development world. Follow the steps very carefully, this is a little tricky but if you do this once, it will increase your git confidence. 

switch to master branch
```
git checkout master
```

add a remote that points to my branch
```
git remote add destiny https://github.com/anishLearnsToCode/hackerrank-math.git
```

check that remotes have been added
```
git remote -v
```

you should see 2 remotes, origin for your branch and destiny for my branch. Now pull from my master branch. (always pull master from me and push master to your branch, as I ma the source of truth `destiny` and you are a copy of my branch `origin`) 

```
git pull destiny master --rebase
``` 

now switch to your feature branch
```
git checkout branch-name
```

and rebase on top of master
```
git rebase master 
```

resolve all conflicts and then continue with rebase
```
git rebase --continue
```

and push to remote `origin`
```
git push --force
```

**Comment by anishLearnsToCode** [comment]
also the solution you provided is incorrect, it doesn't work for all testcases, did you test this before creating the PR? 
please update your solution or I will have to remove it. 
image

**Pull request merged**